### ISSUE TITLE
Full width middle dot

### ISSUE THREAD

**Issue by stone-zeng** [adobe-fonts/source-han-sans#226]
### Prerequisites

* [ ] If you are reporting an issue that affects glyphs for characters for a particular region or regions, did you verify that the characters are within the supported scope of the region or regions? This generally means GB 18030 or Tōngyòng Guīfàn Hànzìbiǎo (通用规范汉字表) for China, Big Five or CNS 11643 Planes 1 & 2 for Taiwan, HKSCS-2016 for Hong Kong, JIS X 0208, JIS X 0212, and JIS X 0213 for Japan, and KS X 1001 and KS X 1002 for Korea.
* [x] Did you thoroughly search the [open](https://github.com/adobe-fonts/source-han-sans/issues) and [closed](https://github.com/adobe-fonts/source-han-sans/issues?q=is%3Aissue+is%3Aclosed) issues to avoid reporting a duplicate issue?
* [x] Did you go through the [official font readme file](https://github.com/adobe-fonts/source-han-sans/raw/release/SourceHanSansReadMe.pdf) to better understand the scope of the project, to include the *Known Issues* section at the very end?

### Description

As stated in [clreq](https://w3c.github.io/clreq), the width of Middle Dot (间隔号/間隔號) U+00B7 MIDDLE DOT [·] is the following:

> Usage of middle dots differs between Traditional Chinese and Simplified Chinese. In principle, the middle dot, either in vertical writing mode or horizontal writing mode, **should have the same dimensions as a character**; however, in Simplified Chinese, the middle dot **sometimes has half the width of a character when it is used to separate the month and the date**, e.g. 9·11.

So please consider to add a full-width version of U+00B7 MIDDLE DOT.

Note that in GB/T 15834-2011 General rules for punctuation (标点符号用法), the width of Middle Dot should have half the width of a character, but the glyph in the standard itself is actually full width (the same as one character).

**Comment by kenlunde** [comment]
The word **sometimes** is key here, and implies that an OpenType feature would need to be invoked in order to access a half-width version of U+00B7 · MIDDLE DOT.

Well, the '[halt](https://docs.microsoft.com/en-us/typography/opentype/spec/features_fj#a-namehalt-idhaltatag-39halt39)' (_Alternate Half Widths_) GPOS features, if invoked, will adjust the horizontal metrics of the glyph for this character such that it becomes precisely half-width. Likewise for vertical, the '[vhal](https://docs.microsoft.com/en-us/typography/opentype/spec/features_uz#a-namevhal-idvhalatag-39vhal39)' (_Alternate Vertical Half Metrics_) GPOS feature, if invoked, will adjust the vertical metrics of the glyph for this character such that it becomes precisely half-width.

In other words, it's not necessary to add a new glyph.

**Comment by stone-zeng** [comment]
We can find that there are three types of middle dots here:

![image](https://user-images.githubusercontent.com/18233088/52493882-89cfc900-2c07-11e9-8656-23b533524517.png)

The full-width middle dots and the half-width one in Chinese is much heavier than the half-width one in Japanese. Is this by design? And how to get the lighter glyph, but with full-width?

By the way, please consider to add this feature in Source Han Serif too.

## Note

I use XeLaTeX to create the figure. The code is here.
<details>
  <summary>LaTeX Code</summary>
  
```latex
\documentclass{article}
\usepackage{fontspec}
\setmainfont{Source Serif Pro}
\setmonofont{Source Code Pro}
\ExplSyntaxOn
\NewDocumentCommand \TEST { m }
  { \section {#1} \__test:n {#1} }
\cs_new:Npn \__test:n #1
  {
    \__test_aux:nn {#1} {pwid} \\
    \__test_aux:nn {#1} {fwid} \\
    \__test_aux:nn {#1} {hwid} \\
    \__test_aux:nn {#1} {palt} \\
    \__test_aux:nn {#1} {halt}
  }
\cs_new:Npn \__test_aux:nn #1#2
  {
    \makebox[2cm][l]{ \texttt{#2} }
    \makebox[2cm][l]{ \fontspec{Source~Han~Serif}[Script=CJK, Language=#1, RawFeature=+#2] 天·地·人 }
    \makebox[2cm][l]{ \fontspec{Source~Han~Sans }[Script=CJK, Language=#1, RawFeature=+#2] 天·地·人 }
  }
\ExplSyntaxOff

\begin{document}
\TEST{Chinese Simplified}
\TEST{Japanese}
\end{document}
```
</details>

**Comment by kenlunde** [comment]
The images above are unhelpful without explicitly indicating the actual code points that are being used. There are a variety of code points that serve as middle dots, such as U+00B7, U+2022, U+2027, U+30FB, and U+FF65. The last one is an explicit half-width character, and may serve your immediate needs.

**Comment by balthild** [comment]
According to the LaTeX code provided, it's U+00B7.

As I tested just now, none of `halt`, `palt`, `pwid`, `hwid`, or `fwid` affects the thickness of U+00B7 at all, in current version of Source Han Sans.
